src/webview/git-import/app/gitImport.tsx

```diff
@@ -236,6 +237,11 @@ export class GitImport extends React.Component<DefaultProps, {
         });
     }
 
+    getTrimmedURL = (value: string): string => {
+        const parsedURL = gitUrlParse(value);
+        return `${parsedURL.protocol}://${parsedURL.source}/${parsedURL.full_name}`;
```

Comment:
If I enter an SSH repo like `git@github.com:datho7561/lemminx.git`, then it gets modified to `ssh:////github.com/datho7561/lemminx`. If I click "Analyze" and select a folder to clone into, it gets stuck at "Cloning the repository" with no visible error message. This is because `ssh:////github.com/datho7561/lemminx` is not the same as `git@github.com:datho7561/lemminx.git`. (You will need to use one of your own repos to test this).

Reply:
Showing error on accessing private repo, Will handle this case on https://github.com/redhat-developer/vscode-openshift-tools/issues/2938
![image](https://github.com/redhat-developer/vscode-openshift-tools/assets/93245779/92379bfe-59e4-499a-9373-be9681a873ae)